cmd/webpkgserver/webpkgserver.example.toml

```diff
@@ -202,6 +200,95 @@
+# is to have only one instance handle automatic certificate renewals and let
+# other instances reload the fresh certificate from the disk. It minimizes the
+# cost (SXG certificates cost money and we don't need each webpackager instance
+# to have a separate SXG certificate) and prevents the webpkgserver instances
+# from bombarding your Certificate Authority with certificate requests. Note
+# this setup requires you to locate the certificate file on a shared filesystem
+# accessible by all webpkgserver instances. Also note that some Certificate
+# Authorities require that purchased certificates be installed on a single
+# (physical) machine.
+#
+# For the full ACME spec, see:
+#   https://tools.ietf.org/html/rfc8555 
+[SXG.ACME]
+  # Enable webpkgserver to attempt to auto renew certificates using
+  # ACME. Required when ACME is enabled.
+  #Enable = false 
+
+  # The ACME discovery URL. It is specified by the Certificate Authority that
+  # doles out your certificates. As of August 2020, only DigiCert supports
+  # automatic renewals of signed exchange certificate:
+  # https://docs.digicert.com/certificate-tools/acme-user-guide/acme-directory-urls-signed-http-exchange-certificates/
+  # Required when ACME is enabled.
+  DiscoveryURL = '<Your Discovery URL>'
+
+  # The email address registered to the Certificate Authority for your signed
+  # exchange certificates.
+  # Required when ACME is enabled.
+  Email = 'user@example.com'
+
+  # For the remaining configuration items, it's important to understand the
+  # different challenges employed as part of the ACME protocol:
+  # https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html#identifier-validation-challenges
+  # https://letsencrypt.org/docs/challenge-types/
+  # https://certbot.eff.org/docs/challenges.html
+  #
+  # Note that you do not need to configure all of these challenges. It is
+  # typically sufficient to configure just one of them. If more than one method
+  # is configured, the go-acme/lego library will decide pick one of them for
+  # primary use and use other settings as backup. For wildcard certificates,
+  # nevertheless, DNSProvider is the only supported method of validation, and
+  # others cannot be used. See DNSProvider for more detail.
+
+  # The http server root directory where the ACME http challenge token should
+  # be deposited. Note that you may need to do some configuration work to
+  # get this setup to work where multiple instances of webpkgserver is running.
+  # For example:
+  # https://community.letsencrypt.org/t/how-to-nginx-configuration-to-enable-acme-challenge-support-on-all-http-virtual-hosts/5622/3
+  HTTPWebRootDir = '/path/to/www_root_dir'
+
+  # The port used by the webpkgserver to respond to the HTTP challenge
+  # issued as part of ACME protocol. Note that if your setup only opens up
+  # certain ports, you may need to do a configuration change where you forward
+  # requests to this port using proxy_pass, for example:
```

Comment:
`proxy_pass` is Nginx-specific. I prefer a more general description as webpkgserver can work with other http servers (e.g. Apache).

Reply:
Clarified the description.